docs/server-manual/links/yamcs-cascading.rst

```diff
@@ -107,3 +107,23 @@ connectionAttempts (integer)
 
 reconnectionDelay (integer)
    If the connection fails or breaks, the time (in milliseconds) to wait before reconnection.
+
+CommandMapping (list of CommandMapData)
```

Comment:
shouldn't this be "commandMapping" (starting with lower case like the other ones)?

If not specified, will it work like before (it should)? Then can you write that (also the equivalent config)?